runner.py

```diff
@@ -274,6 +274,25 @@ def process_data(with_indels, dust, evalue, fastq_files, log, max_basecall_itera
     return reference_file
 
 
+def validate_input(output_dir, input_dir, reference_file):
+    if os.path.exists(output_dir):
+        list_dir = os.listdir(output_dir)
+        if len(list_dir) > 0:
+            if not ((len(list_dir) == 1) and (list_dir[0] == 'pbs_logs')):
+                raise Exception("output_dir must be path to a new or empty directory!")
+    if not os.path.isfile(reference_file):
+        raise Exception("Reference_file must exist!")
+    else:
+        if not os.path.basename(reference_file).endswith(".fasta"):
+            raise Exception("Reference_file must end with .fasta!")
+    if not os.path.isdir(input_dir):
+        raise Exception("Input_dir must exist!")
+    files_fasta = get_files_by_extension(input_dir, "fastq")
+    files_fastagz = get_files_by_extension(input_dir, "fastq.gz")
+    if len(files_fasta) == 0 and len(files_fastagz) == 0:
+        raise Exception("There are no files ending with fastq or gz in input_dir!")
```

Comment:
should still be a bit more accurate:
"Could not find files ending with '.fastq' or 'fastq.gz' in input_dir !"